README.md

```diff
@@ -243,6 +242,56 @@ In the strict mode, `async` functions are checked such that they only raise
 effects on forward declarations, callbacks and methods using
 `{.raises: [CatchableError].}` (or more strict) annotations.
 
+### Cancellation support
+
+Any running `Future` can be cancelled.
+
+```nim
+proc c1 {.async.} =
+  echo "Before sleep"
+  try:
+    await sleepAsync(10.minutes)
+    echo "After sleep" # never reached
+  except CancelledError as exc:
+    echo "We got cancelled!"
+    raise exc
+
+proc c2 {.async.} =
+  await c1()
+  echo "Never reached, since the CancelledError bubble up"
```

Comment:
This `echo` will never be printed just because of `c1`, but if `c1`, but this code depends on `c1` code. If `c1` code will not have any continuations this `echo` will be printed.